### PR TITLE
JBIDE-19022: JBoss Forge New and Noteworthy for 4.2.2.Final

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

File: documentation/whatsnew/forge/forge-news-4.2.2.Final.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+:page-product_id: jbt_core 
+:page-product_version: 4.2.2.Final
+
+== Forge 2 Runtime 	
+
+The included Forge 2 runtime is 2.13.0.Final. See the release notes link:http://forge.jboss.org/news/2014-12-15/forge-2.13.0.final-%28flange%29-is-here[here]
+
+== Command window is now focused on Linux environments
+
+When the command window is invoked (Ctrl + 4), the command window was not being focused when run in Linux.
```
Comment (xcoulon) [inline documentation/whatsnew/forge/forge-news-4.2.2.Final.adoc]: do you have a related JIRA for that bug ?

Comment (gastaldi) [inline documentation/whatsnew/forge/forge-news-4.2.2.Final.adoc]: Yes, it's JBIDE-18650

**Comment by gastaldi** [comment]
Anything missing in this PR?

**Comment by xcoulon** [comment]
@gastaldi sorry for the delay, I thought you would update the PR with a link to the related JIRA. 
I just did it myself (and I took the liberty to reword a little bit the first paragraph). Please let me know if this is OK for you (see the page on the staging site: http://tools-stg.jboss.org/documentation/whatsnew/jbosstools/4.2.2.Final.html) and I'll push it on the main site afterwards.
Thanks.

**Comment by gastaldi** [comment]
Looks fantastic! Thanks @xcoulon

**Comment by xcoulon** [comment]
@gastaldi cool! and again, sorry for the delay :/

**Pull request closed without merging**